app/views/shoes/show.html.erb

```diff
@@ -1,33 +1,27 @@
-<%# <%= render 'navbar' %> %>
+<%= render 'pages/navbar' %>
```

Comment:
@LamoG6 this can be removed as it is now in the main appplication html file

Reply:
sure will do